lib/engine/game/g_18_mex/game.rb

```diff
@@ -1090,6 +1090,17 @@ def train_limit(entity)
           super + Array(abilities(entity, :train_limit)).sum(&:increase)
         end
 
+        def action_processed(action)
+          case action
```

Comment:
Should it not be super first, and then case?

Reply:
Hmm maybe it should. I was aping how 18Chesapeake does `action_processed`, which also doesn't call `super`. In `Game::Base`, `action_processed` just checks for corporations to close, which doesn't apply to Chesapeake or MEX, so it'll work without this, but we'll run into trouble if `Game::Base#action_processed` gets more functionality put in.